judgments/tests/test_search.py

```diff
@@ -194,6 +195,74 @@ def test_search_breadcrumbs_with_query_string(
     ):
         mock_search_judgments_and_parse_response.return_value = FakeSearchResponse()
         response = self.client.get("/judgments/search?query=waltham+forest")
+
         assert response.context["breadcrumbs"] == [
             {"text": 'Search results for "waltham forest"'}
         ]
+
+
+class TestSearchFacets(TestCase):
+    @patch("judgments.views.advanced_search.api_client")
+    @patch("judgments.views.advanced_search.search_judgments_and_parse_response")
+    def test_populated_court_facets(
+        self, mock_search_judgments_and_parse_response, mock_api_client
+    ):
+        mock_search_judgments_and_parse_response.return_value = FakeSearchResponse()
+        eat_court_code = all_courts.get_by_code("EAT")
+        response = self.client.get("/judgments/search?query=example+query")
+
+        # Desired court_facet is present
+        assert response.context["context"]["court_facets"] == {eat_court_code: "3"}
+        # Blank keys are not present
+        assert {"": "3"}.items() not in response.context["context"][
```

Comment:
I'd probably write `assert "" not in complicated_dict.keys()`; this will always pass because the value of `""` is `5`.